examples/weather/unified_recipe/README.md

```diff
@@ -84,3 +84,14 @@ training, run
 
 If running inside a docker container, you may need to include the
 `--allow-run-as-root` in the multi-GPU run command.
+
+## SFNO Training
+
+One of the showcased models available in the configs is [Spherical Fourier Neural Operators:
+Learning Stable Dynamics on the Sphere](https://arxiv.org/pdf/2306.03838.pdf).
+
+The config file can be modified to train the SFNO model by uncommenting all SFNO configs. Following the prior dataset fetching and 
```

Comment:
"Following the prior dataset fetching and" - this is sounding incomplete sentence... Am I interpreting this correctly?

Reply:
Whoops, Finished the configs with info on install Makani. Tested it out and it runs. I would really have liked to make SFNO the default but I decided to keep a tiny AFNO model for now.